projects/plugins/crm/tests/action-test-php.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eo pipefail
+
+if [[ -z "$CI" ]]; then
+	echo "This script is only for use in the CI environment. Sorry."
+	exit 1
+fi
+
+: "${WORDPRESS_DIR:?WORDPRESS_DIR needs to be set and non-empty.}"
+
+# Setup database. Even though tests/acceptance.suite.yml contains commands to create it, it chokes before it gets to run them without this.
+mysql -e "DROP DATABASE IF EXISTS jpcrm_testing; CREATE DATABASE jpcrm_testing;"
+
+# Setup config.
+cp tests/acceptance.suite.dist.yml tests/acceptance.suite.yml
+sed -i 's/some_db_user/root/g' tests/acceptance.suite.yml
+sed -i 's/some_db_pass/root/g' tests/acceptance.suite.yml
+sed -i 's!/path/to/test/file/must-overwrite-it-in-acceptance.suite.yml!'"$WORDPRESS_DIR"'!g' tests/acceptance.suite.yml
+
+# Setup WordPress runtime config.
+cp "$WORDPRESS_DIR/wp-config-sample.php" "$WORDPRESS_DIR/wp-config.php"
+sed -i 's/database_name_here/jpcrm_testing/g' "$WORDPRESS_DIR/wp-config.php"
+sed -i 's/username_here/root/g' "$WORDPRESS_DIR/wp-config.php"
+sed -i 's/password_here/root/g' "$WORDPRESS_DIR/wp-config.php"
+sed -i '/Add any custom values between this line and the "stop editing" line./a define("DISABLE_WP_CRON", true);\ndefine("WP_AUTO_UPDATE_CORE", false);' "$WORDPRESS_DIR/wp-config.php"
+
+# Build and run tests
+composer build-tests
+composer tests
```

Comment:
If we want to merge this to get the infrastructure there and then make the tests pass in a later PR, we can temporarily ignore failures here by doing like
```suggestion
composer tests || true
```
Be sure to revert that once tests pass, of course.